File: DTOs/MessageDTO.cs
namespace Mint.API.DTOs
{
    public interface MessageDTO
	{

		int? Id { get; set; }
		string Title { get; set; }
		bool IsDraft { get; set; }
		string? Header { get; set; }
		string? Body { get; set; }
		string? Background { get; set; }
		DateTime? StartDate { get; set; }
		DateTime? EndDate { get; set; }
		int? Icon { get; set; }
		int? Action { get; set; }
		List<int> Categories { get; set; }

	}
}

File: client/src/app/message/interfaces/Message.dto.ts
import { Action } from './Action';
import { Category } from './Category';
import { Icon } from './Icon';

export interface MessageDTO {

  id?: number;
  isDraft: boolean;
  title: string;
  header: string | null;
  body: string | null;
  background: string | null;
  startDate: Date | string | null;
  endDate: Date | string | null;
  icon: number | null;
  action: number | null;
  categories: number[];

}
